R/object_usage_linter.R

```diff
@@ -197,7 +199,7 @@ get_assignment_symbols <- function(xml) {
 }
 
 parse_check_usage <- function(expression, known_used_symbols = character(), declared_globals = character(),
-                              start_line = 1L) {
+                              start_line = 1L, skip_with = FALSE) {
```

Comment:
any reason not to change the default here too?

Reply:
No reason; just a leftover from the original approach I had taken in the PR. 

Changed to `skip_with  = TRUE`.